fixtures/env.test-agents

```diff
@@ -0,0 +1,29 @@
+# Agent environment var overrides
+
+KATHY_CHAT_MESSAGES="Chat message 1,Chat message 2,Chat message 3"
+KATHY_INTERVAL=999
+KATHY_ENABLED=true
+
+PROCESSOR_ALLOWED=0x1111111111111111111111111111111111111111111111111111111111111111,0x1111111111111111111111111111111111111111111111111111111111111112,0x1111111111111111111111111111111111111111111111111111111111111113
+PROCESSOR_DENIED=0x1111111111111111111111111111111111111111111111111111111111111111,0x1111111111111111111111111111111111111111111111111111111111111112,0x1111111111111111111111111111111111111111111111111111111111111113
+PROCESSOR_SUBSIDIZED_REMOTES=chain1,chain2,chain3
+PROCESSOR_S3_BUCKET=aws-bucket
+PROCESSOR_S3_REGION=region-1
```

Comment:
The approach here was to expose the entirety of the config struct. Are we 100% no one (internal or external) will ever want to override this? Also, I'm not sure if it makes sense to trim it since it's already in there.

Reply:
Ok sounds good. Since, it's not required, it makes sense to keep this feature around